modules/mono/editor/GodotTools/GodotTools/Build/DotNetFinder.cs

```diff
@@ -55,7 +56,8 @@ public static string FindDotNetExe()
             process.StartInfo = new ProcessStartInfo(dotNetExe, "--list-sdks")
             {
                 UseShellExecute = false,
-                RedirectStandardOutput = true
+                RedirectStandardOutput = true,
+                StandardOutputEncoding = Encoding.UTF8,
```

Comment:
Not setting the Error encoding here, because the error stream is not redirected.

Reply:
Since we set `DOTNET_CLI_UI_LANGUAGE` to `en-US` for this process. Is UTF-8 needed here?

Reply:
a) `--list-sdks` doesn't write any localized string, so the language actually never matters: https://github.com/dotnet/runtime/blob/b4b3105e944409ad0a20eb2c1948cf98ef2833f1/src/native/corehost/fxr/sdk_info.cpp#L87-L97
b) The encoding still can matter if for some godforsaken reasons someone installed an sdk at a path containing non-ansi chars
c) I mostly kept it for consistency